SECP-service/src/main/java/com/visucius/secp/resources/GroupResource.java

```diff
@@ -0,0 +1,32 @@
+package com.visucius.secp.resources;
+
+
+import com.codahale.metrics.annotation.Timed;
+import com.visucius.secp.Controllers.GroupController;
+import com.visucius.secp.DTO.GroupCreationRequest;
+import io.dropwizard.auth.Auth;
+import io.dropwizard.hibernate.UnitOfWork;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+@Path("/")
+public class GroupResource {
+
+    private GroupController groupController;
+
+    public GroupResource(GroupController controller)
+    {
+        this.groupController = controller;
+    }
+
```

Comment:
What if I want to modify the roles or permission for a group. I don't see any functionality for that here.